### PR TITLE
ARM ShazBots

### PR DESCRIPTION
**Pull request by Doloreere**
## Details
<details>
  <summary>Expand</summary>
-added ShazBot's sprites for ARM, now only green 
</details>

### CODE REVIEW

**Comment by Garen7** [comment]
You have some undocumented changes to backpack/satchel icons, I'm not sure if you meant to have that in your PR but if you don't fix the icon_state variables of the satchels/backpacks you're trying to change then you wont be able to see them on the character anymore.

Also if you have trouble with merge conflicts for icons or maps then you can setup the precommit hooks to help prevent them by running tools/hooks/install.bat on windows or tools/hooks/install.sh on unix.

**Comment by SpadesSlick** [comment]
What's the reason of having multiple colors for one gun?

**Comment by Doloreere** [comment]
> What's the reason of having multiple colors for one gun?

deleted all version instead of grey for traitors.

File: code/modules/projectiles/guns/projectile/sniper.dm [hunk 1]
```diff
@@ -19,13 +19,30 @@
 	matter = list(MATERIAL_PLASTEEL = 40, MATERIAL_PLASTIC = 20)
 	price_tag = 5000
 	var/bolt_open = 0
+	var/item_suffix = ""
 	zoom_factor = 2.0
 
+/obj/item/weapon/gun/projectile/heavysniper/ironhammer
```
Comment (SpadesSlick) [inline code/modules/projectiles/guns/projectile/sniper.dm]: Why do we need Ironhammer version?

Comment (Doloreere) [reply code/modules/projectiles/guns/projectile/sniper.dm]: forgot to delete it

Comment (Doloreere) [reply code/modules/projectiles/guns/projectile/sniper.dm]: forgot to delete it

**Pull request merged**